homeassistant/components/smlight/strings.json

```diff
@@ -68,6 +68,32 @@
       },
       "socket_uptime": {
         "name": "Zigbee uptime"
+      },
+      "device_ip": {
+        "name": "IP address"
+      },
+      "device_mode": {
+        "name": "Connection mode",
+        "state": {
+          "0": "Ethernet",
+          "1": "Wi-Fi",
+          "2": "USB"
```

Comment:
Oh you already do, but can we get a better state than just a number? This is useful for templating and stuff

Reply:
yes, can lookup values.